funnel/templates/macros.html.jinja2

```diff
@@ -51,7 +51,7 @@
     {{ form.hidden_tag() }}
     <p class="no-jshidden zero-bottom-margin">
       {%- if current_auth.is_anonymous %}
-        <a class="mui--text-body1"  href="{{ url_for('login') }}" data-action="Login to save project">{{ faicon(icon='bookmark', css_class='mui--text-light') }}</a>
+        <a class="mui--text-body1"  href="{{ url_for('login') }}" data-action="Login to save project"><span class="mui--text-body1 mui--text-light login-txt">{% trans %}Login <span class="mui--hidden-xs mui--hidden-sm">to save</span>{% endtrans %}</span>{{ faicon(icon='bookmark', css_class='mui--text-light') }}</a>
```

Comment:
Make this two separate strings, or the translator will be looking at `mui-hidden-xs`